bin/parse

```diff
@@ -36,6 +37,6 @@ else
   parts.each_with_index do |(key, value), index|
     puts if index > 0
     puts "#{key}:"
-    puts value.inspect
+    pp value
```

Comment:
This prints the warnings much more nicely, one per line.

OTOH it shows DATA as `(6,0)-(11,0)` instead of `#<Prism::Location @start_offset=90 @length=215 start_line=6>`.
That seems an OK trade-off to keep things simple.